realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -192,7 +192,7 @@ public static synchronized void init(Context context) {
             defaultConfiguration = new RealmConfiguration.Builder(context).build();
             ObjectServerFacade.getSyncFacadeIfPossible().init(context);
             BaseRealm.applicationContext = context.getApplicationContext();
-            SharedRealm.initialize(new File(context.getFilesDir(), ".realmNamedPipes"));
+            SharedRealm.initialize(new File(context.getFilesDir(), ".realm.temp"));
```

Comment:
This is the doc for it: https://developer.android.com/reference/android/content/Context.html#getCacheDir() 

Which doesn't really answer the question though. My main concern is what happens if a tmp file is backed up and restored I think.